api/src/main/java/com/codeforcommunity/dto/school/SchoolListResponse.java

```diff
@@ -0,0 +1,30 @@
+package com.codeforcommunity.dto.school;
+
+import java.util.List;
+
+public class SchoolListResponse {
+
+  private int count;
+  private List<School> schools;
```

Comment:
Curious what the benefit of returning count is. How inefficient is `.length` in JavaScript?

Reply:
Uhh I don't know I just figured if we we're working with 900+ schools it might be worth it?